app/controllers/profiles_controller.rb

```diff
@@ -79,6 +80,8 @@ def munge_tag_string
   end
 
   def profile_params
-    params.require(:profile).permit(:first_name, :last_name, :gender, :bio, :location, :searchable, :tag_string, :nsfw, :date => [:year, :month, :day]) || {}
+    params.require(:profile).permit(:first_name, :last_name, :gender, :bio,
+                                    :location, :searchable, :tag_string, :nsfw,
+                                    :public_details, :date => %i(year month day)) || {}
```

Comment:
Please update this to the Ruby 1.9 hash syntax while touching, `date: %i(year month day)`